src/components/sequencing/Sequences.svelte

```diff
@@ -51,7 +51,7 @@
     workspaceId = event.detail;
 
     if (browser) {
-      setQueryParam(SearchParameters.WORKSPACE_ID, `${workspaceId}` ?? null);
+      setQueryParam(SearchParameters.WORKSPACE_ID, `${workspaceId ?? null}`);
```

Comment:
I think the way it was before is correct. This will result in a `"null"` being present in the URL.

Reply:
`null` for a workspace id isn't valid, and it looks like the null check inside the function will delete the key from the URL which is what we want?